### ISSUE TITLE
After I upload a custom card. ioBroker crashed.

### ISSUE THREAD

**Issue by SchumyHao** [ioBroker/ioBroker.lovelace#40]
I update a custom card by drop a js on web page.
Then admin start abnormal

issue logs shows
```
host.dockerhub-cn Cannot get view: ReplyError: ERR Error running script (call to f_47ca5e051ba19850d94c45a1fc8725ff04ae868f): @user_script:12: user_script:12: Expected value but found T_END at character 1 
```

Then I restart ioBroker system. All adapters can not work

I delete lovelace adapter by commad line. This issue still can not fix.

I run `iob list instances`
An issue log shows
```
Cannot parse JSON cfg.o.system.adapter.lovelace.0: 
```

**Comment by SchumyHao** [comment]
It reproduceable.

1. install lovecace.
2. enable authtation and history (Not sure must or not)
3. restart lovelace instance.
4. upload a custom card js. I upload this https://github.com/custom-cards/bignumber-card/blob/master/bignumber-card.js
5. click Restart Adapter to upload files
6. issue happened

I run iob in docker, redis/redis databese, using js-controller 2.0.25

**Comment by SchumyHao** [comment]
The way to fix this 
![image](https://user-images.githubusercontent.com/4982108/66717230-2c7cbf00-ee09-11e9-9a45-2a5c9404f68c.png)

**Comment by SchumyHao** [comment]
@Apollon77 helped me to analysis this issue. And he update js-controller to 2.0.27.

After I update js-controller to 2.0.27, Then I press Restart Adapter to upload files button.
Some error logs shows

```
2019-10-13 23:11:50.165  - warn: admin.0 (36) setObject: Argument object is no object: function(){
    // prevent double callbacks
    if (sent) return;
    var args = Array.prototype.slice.call(arguments);
    debug('sending ack %j', args);

    var type = hasBin(args) ? parser.BINARY_ACK : parser.ACK;
    self.packet({
      id: id,
      type: type,
      data: args
    });

    sent = true;
  }
```

Maybe this info is useful

**Comment by Apollon77** [comment]
Controller noch prevents setting objects that are no objects.

**Comment by Apollon77** [comment]
@GermanBluefox It seems that lovelace tries to set the instanceObject "system.adaper.lovelace.0" with the above string as value ... I more expect that it is a function. Do you have any idea where and why this happens?

**Comment by GermanBluefox** [comment]
Fixed in 0.2.1
